### PR TITLE
WinUI-2539 : Prepare UG documentation for shadow control in WinUI.

### PR DESCRIPTION
**Pull request by Priyanka-Vijayasankar**
### Description
Prepare UG documentation for shadow control in WinUI.

### CODE REVIEW

File: winui-toc.html [hunk 1]
```diff
@@ -1,75 +1,60 @@
...
 <ul class="treeview">
     <li>
-		<a href="/cr/winui" target="_blank">API Reference</a>
-	</li>
-    <li>
-		<a href="/winui/overview">Overview</a>
-	</li>
-	<li>
-	   <a href="/winui/System-Requirements">System Requirements</a>
-	</li>
```
Comment (Manivannan-E) [inline winui-toc.html:11]: @Priyanka-Vijayasankar Remove unwanted changes.

File: winui/Shadow/Overview.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: What's New for .NET WinUI Shadow Control | Syncfusion
+description: The .NET WinUI Shadow control is used to apply shadow effects to any framework element to create a beautiful and appealing UI.
+platform:winui-controls
+control: SfShadow
+documentation: ug
+---
+
+# WinUI Shadow (New)
```
Comment (Manivannan-E) [inline winui/Shadow/Overview.md:10]: @Priyanka-Vijayasankar WinUI Shadow Overview.

File: winui/Shadow/Overview.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+The [WinUI Shadow](https://www.syncfusion.com/winui-controls/shadow) .NET WinUI Shadow control is used to apply shadow effects to framework elements to give them an appearance of depth, allowing users to easily differentiate overlapping elements and enjoy a beautiful and appealing user interface.
+
+## Shadow control
```
Comment (Manivannan-E) [inline winui/Shadow/Overview.md:14]: @Priyanka-Vijayasankar Remove the header ## Shadow control

File: winui/Shadow/Overview.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+![Displaying the Shadow control]()
+
+## Key features
+
+* Shadow Color: Applies the user's own color to the `shadow`.
+* Shadow BlurRadius: Applies a blur radius for the `shadow`.
+* Shadow Offset: `OffsetX` and `OffsetY` properties, position the shadow relative to the position of the view.
+* Shadow CornerRadius: SfShadow provide CornerRadius property, you can soften the `shadow` edges and make them look more essential.
```
Comment (Manivannan-E) [inline winui/Shadow/Overview.md:23]: @Priyanka-Vijayasankar Applies the corner radius for the shadow.

File: winui/Shadow/Getting-Started.md [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+title: Getting Started with WinUI Shadow control | Syncfusion
+description: Learn here all about getting started with Syncfusion WinUI Shadow(SfShadow) control, its elements, and more.
+platform: winui-controls
+control: SfShadow
+documentation: ug
+---
+
+# Getting Started with WinUI Shadow
+
+This section explains the steps required to add the [WinUI Shadow](https://www.syncfusion.com/winui-controls/shadow) control. This section covers only basic features needed to get started with Syncfusion `Shadow` control.
```
Comment (Manivannan-E) [inline winui/Shadow/Getting-Started.md:12]: @Priyanka-Vijayasankar This section explains the steps required to add the WinUI Shadow control and covers only basic features needed to get started with Syncfusion `SfShadow` control.

File: winui/Shadow/Getting-Started.md [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+    /// </summary>
+    public sealed partial class MainPage : Page
+    {
+        public MainPage()
+        {
+           this.InitializeComponent();
+           // Creating an instance of the Shadow control.
+           SfShadow shadow = new SfShadow();
+
+           //Setting Color, BlurRadius, OffsetX, OffsetY, ShadowCornerRadius of Shadow.
```
Comment (Manivannan-E) [inline winui/Shadow/Getting-Started.md:66]: @Priyanka-Vijayasankar Add Content code in C#.

File: winui/Shadow/Getting-Started.md [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+    xmlns:local="using:GettingStarted"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:syncfusion="using:Syncfusion.UI.Xaml.Core"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    <Grid>
+       <syncfusion:SfShadow Name="buttonShadow"
+                      Color="#40000000"
+                      BlurRadius="8"
```
Comment (Manivannan-E) [inline winui/Shadow/Getting-Started.md:38]: @Priyanka-Vijayasankar Remove the shadow control properties.

File: winui/Shadow/Getting-Started.md [hunk 8]
```diff
@@ -0,0 +1,181 @@
...
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+![Shadow added to the Button Control](Shadow_images/Button.png)
+
+
+## Adding Image for Shadow
+
```
Comment (Manivannan-E) [inline winui/Shadow/Getting-Started.md:83]: @Priyanka-Vijayasankar Applying shadow effect for Image

File: winui/Shadow/Getting-Started.md [hunk 9]
```diff
@@ -0,0 +1,181 @@
...
+           shadow.OffsetY = 4;
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+
+![Shadow added to the Image](Shadow_images/Image.png)
```
Comment (Manivannan-E) [inline winui/Shadow/Getting-Started.md:129]: @Priyanka-Vijayasankar Follow image name properly with platform name. for eg: winui_shadow_image.

File: winui/Shadow/Customization.md [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+control: SfShadow
+documentation: ug
+---
+
+# Customization in WinUI Shadow
+
+This section explains the customization features available in the WinUI [Shadow](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Core.SfShadow.html) control.
+
+## Custom colors for displaying the Shadow for button control
+
```
Comment (Manivannan-E) [inline winui/Shadow/Customization.md:15]: @Priyanka-Vijayasankar Applying color for shadow.

File: winui/Shadow/Customization.md [hunk 11]
```diff
@@ -0,0 +1,146 @@
...
+           shadow.Color = Color.FromArgb(125, 255, 0, 0);
+           shadow.BlurRadius = 8;
+           shadow.OffsetX = 4;
+           shadow.OffsetY = 4;
+        }
+    }
+}
+
+![Displaying the Shadow with custom colors](Shadow_images/ButtonColor.png)
+
```
Comment (Manivannan-E) [inline winui/Shadow/Customization.md:55]: @Priyanka-Vijayasankar  Color customization in WinUI Shadow.

File: winui/Shadow/Overview.md (hunk 2 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Overview.md:10]: Removed new word in WinUI Shadow Overview title.

File: winui/Shadow/Overview.md (hunk 3 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Overview.md:14]: Removed the header.

File: winui/Shadow/Overview.md (hunk 4 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Overview.md:23]: Change the content in corner radius as mentioned above.

File: winui/Shadow/Getting-Started.md (hunk 5 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Getting-Started.md:12]: Changed the content as mentioned above.

File: winui/Shadow/Getting-Started.md (hunk 6 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Getting-Started.md:66]: Added content code in C#.

File: winui/Shadow/Getting-Started.md (hunk 7 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Getting-Started.md:38]: Removed the shadow control properties.

File: winui/Shadow/Getting-Started.md (hunk 9 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Getting-Started.md:129]: Followed the image name as mentioned above.

File: winui/Shadow/Customization.md (hunk 11 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Customization.md:55]: Changed the content as mentioned above.

File: winui/Shadow/Customization.md [hunk 12]
```diff
@@ -0,0 +1,134 @@
...
+documentation: ug
+---
+
+# Customization in WinUI Shadow
+
+This section explains the customization features available in the WinUI [Shadow] control.
+
+## Applying Color for shadow
+
+If you want to change the color of the shadow other than the default Color value, use the `Color` property. The default value of `Color` property is `Black` color with 25% alpha value.
```
Comment (Manivannan-E) [inline winui/Shadow/Customization.md:16]: @Priyanka-Vijayasankar The shadow color can be customized using the `Color` property. The default value of `Color` property is `Black` color with 25% alpha value.

File: winui/Shadow/Customization.md [hunk 13]
```diff
@@ -0,0 +1,134 @@
...
+   shadow.Content = button;
+   shadow.Height = 50;
+   shadow.width = 100;
+   shadow.Content = "Button";
+
+{% endhiglight %}
+{% endtabs %}
+
+![Color customization in shadow control in WinUI](Shadow_images/winui_shadow_color.png)
+
```
Comment (Manivannan-E) [inline winui/Shadow/Customization.md:44]: @Priyanka-Vijayasankar Color customization in WinUI Shadow control.

File: winui/Shadow/Customization.md [hunk 14]
```diff
@@ -0,0 +1,134 @@
...
+## Applying BlurRadius for shadow
+
+If you want to change the blur radius of the shadow other than the default value, use the `BlurRadius` property. The default value of `BlurRadius` property is 8.
```
Comment (Manivannan-E) [inline winui/Shadow/Customization.md:47]: @Priyanka-Vijayasankar Blur level of shadow effect can be customized using the `BlurRadius` property. The default value of `BlurRadius` property is 8.

File: winui/Shadow/Customization.md [hunk 15]
```diff
@@ -0,0 +1,134 @@
...
+
+{% tabs %}
+{% highlight XAML %}
+
+<syncfusion:SfShadow>
+    <Button Height="50" width="100" Content="Button"
```
Comment (Manivannan-E) [inline winui/Shadow/Customization.md:53]: @Priyanka-Vijayasankar Change the code based on the output.

File: winui/Shadow/Customization.md [hunk 16]
```diff
@@ -0,0 +1,134 @@
...
+   shadow.Height = 50;
+   shadow.width = 100;
+   shadow.Content = "Button";
+
+{% endhiglight %}
+{% endtabs %}
+
+![BlurRadius customization in shadow control in WinUI](Shadow_images/winui_shadow_blurradius.png)
+
+## Applying Offset for shadow
```
Comment (Manivannan-E) [inline winui/Shadow/Customization.md:75]: @Priyanka-Vijayasankar ## Positioning the shadow.

The position of shadow can be customized using the OffsetX and OffsetY properties. The default value of `OffsetX` and `OffsetY` properties is 4.

File: winui/Shadow/Customization.md [hunk 17]
```diff
@@ -0,0 +1,134 @@
...
+   shadow.Content = button;
+   shadow.Height = 50;
+   shadow.width = 100;
+   shadow.Content = "Button";
+
+{% endhiglight %}
+{% endtabs %}
+
+![OffsetX and OffsetY position customization in shadow control in WinUI](Shadow_images/winui_shadow_offset.png)
+
```
Comment (Manivannan-E) [inline winui/Shadow/Customization.md:105]: @Priyanka-Vijayasankar Positioning in WinUI Shadow control.

File: winui/Shadow/Customization.md [hunk 18]
```diff
@@ -0,0 +1,134 @@
...
+## Applying CornerRadius for shadow
+
+If you want to change the corner radius of the shadow other than the default value, use the `CornerRadius` property. The default value of `CornerRadius` property is 0.
+
```
Comment (Manivannan-E) [inline winui/Shadow/Customization.md:109]: @Priyanka-Vijayasankar The corner radius of the shadow can be customized using the `CornerRadius` property. The default value of `CornerRadius` property is 0.

File: winui/Shadow/Getting-Started.md [hunk 19]
```diff
@@ -0,0 +1,180 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+![Shadow added to the Button Control](Shadow_images/winui_shadow_button.png)
+
+
+## Applying Shadow effect for image
+
+If you can apply the shadow effect for any type of image with the help of shadow control.
+
```
Comment (Manivannan-E) [inline winui/Shadow/Getting-Started.md:67]: @Priyanka-Vijayasankar You can apply the shadow effects for any types of `Image` with the help of `SfShadow` control.

File: winui/Shadow/Getting-Started.md [hunk 20]
```diff
@@ -0,0 +1,180 @@
...
+     BitmapImage bitmapImage = new BitmapImage();
+     bitmapImage.UriSource = new Uri("ms-appx:///Assets/Image/Ellipse_Shadow.png");
+     image.Source = bitmapImage;
+
+{% endhighlight %}
+{% endtabs %}
+
+
+![Shadow added to the Image](Shadow_images/winui_shadow_image.png)
+
```
Comment (Manivannan-E) [inline winui/Shadow/Getting-Started.md:94]: @Priyanka-Vijayasankar Remove unwanted line space.

File: winui/Shadow/Getting-Started.md [hunk 21]
```diff
@@ -0,0 +1,180 @@
...
+
+## Applying Shadow effect for shape
+
+If you can apply the shadow effect for any type of shapee with the help of shadow control.
+
```
Comment (Manivannan-E) [inline winui/Shadow/Getting-Started.md:99]: @Priyanka-Vijayasankar  You can apply the shadow effects for any types of `Shape` and `Path` with the help of SfShadow control.

File: winui/Shadow/Getting-Started.md [hunk 22]
```diff
@@ -0,0 +1,180 @@
...
+{% tabs %}
+{% highlight XAML %}
+
+Viewbox Height="300" Width="300">
+   <StackPanel Orientation="Horizontal">
+       <syncfusion:SfShadow>
```
Comment (Manivannan-E) [inline winui/Shadow/Getting-Started.md:105]: @Priyanka-Vijayasankar Remove the ViewBox tag.

File: winui/Shadow/Customization.md (hunk 12 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Customization.md:16]: Changed the color content as mentioned above.

File: winui/Shadow/Customization.md (hunk 13 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Customization.md:44]: Changed the content as mentioned above.

File: winui/Shadow/Customization.md (hunk 14 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Customization.md:47]: Changed the BlurRadius content as mentioned above.

File: winui/Shadow/Customization.md (hunk 15 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Customization.md:53]: Changed the code based on the output.

File: winui/Shadow/Customization.md (hunk 16 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Customization.md:75]: Changed the offset content as mentioned above.

File: winui/Shadow/Customization.md (hunk 17 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Customization.md:105]: Changed the content as mentioned above.

File: winui/Shadow/Customization.md (hunk 18 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Customization.md:109]: Changed the CornerRadius content as mentioned above.

File: winui/Shadow/Getting-Started.md [hunk 23]
```diff
@@ -0,0 +1,180 @@
...
+description: Learn here all about getting started with Syncfusion WinUI Shadow(SfShadow) control, its elements, and more.
+platform: winui-controls
+control: SfShadow
+documentation: ug
+---
+
+# Getting Started with WinUI Shadow
+
+This section explains the steps required to add the WinUI Shadow control and covers only basic features needed to get started with Syncfusion SfShadow control.
+
```
[the remark this answers is not in the document]
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Getting-Started.md:13]: Highlighted the SfShadow.

File: winui/Shadow/Getting-Started.md (hunk 19 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Getting-Started.md:67]: Changed the image content as mentioned above.

File: winui/Shadow/Getting-Started.md (hunk 20 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Getting-Started.md:94]: Removed unwanted line space.

File: winui/Shadow/Getting-Started.md (hunk 21 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Getting-Started.md:99]: Changed the shape content as mentioned above.

File: winui/Shadow/Getting-Started.md (hunk 22 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Getting-Started.md:105]: Removed the viewbox tag.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

File: winui/Shadow/Customization.md [hunk 24]
```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title: Customization in WinUI Shadow control | Syncfusion
+description: Learn here all about the customization features such as color, opacity, position, and blur radius in Syncfusion WinUI Shadow control.
```
Comment (OnyimbiNelson) [inline winui/Shadow/Customization.md:4]: description: Learn all about the customization features such as color, opacity, position, and blur radius in Syncfusion's WinUI Shadow control here.

File: winui/Shadow/Customization.md [hunk 25]
```diff
@@ -0,0 +1,130 @@
...
+documentation: ug
+---
+
+# Customization in WinUI Shadow
+
+This section explains the customization features available in the WinUI Shadow control.
+
+## Applying Color for shadow
+
+The shadow color can be customized using the `Color` property. The default value of `Color` property is `Black` color with 25% alpha value.
```
Comment (OnyimbiNelson) [inline winui/Shadow/Customization.md:16]: The shadow color can be customized using the `Color` property. The default value of the `Color` property is `Black` color with a 25% alpha value.

File: winui/Shadow/Customization.md [hunk 26]
```diff
@@ -0,0 +1,130 @@
...
+   shadow.Content = button;
+
+{% endhiglight %}
+{% endtabs %}
+
+![Color customization in WinUI Shadow control](Shadow_images/winui_shadow_color.png)
+
+## Applying BlurRadius for shadow
+
+Blur level of shadow effect can be customized using the `BlurRadius` property. The default value of `BlurRadius` property is 8.
```
Comment (OnyimbiNelson) [inline winui/Shadow/Customization.md:44]: The blur level of the shadow effect can be customized using the `BlurRadius` property. The default value of `BlurRadius` property is 8.

File: winui/Shadow/Getting-Started.md [hunk 27]
```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: Getting Started with WinUI Shadow control | Syncfusion
+description: Learn here all about getting started with Syncfusion WinUI Shadow(SfShadow) control, its elements, and more.
```
Comment (OnyimbiNelson) [inline winui/Shadow/Getting-Started.md:4]: description: Learn all about getting started with the Syncfusion WinUI Shadow(SfShadow) control, its elements, and more here.

File: winui/Shadow/Customization.md (hunk 24 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Customization.md:4]: Changed the description as mentioned above.

File: winui/Shadow/Customization.md (hunk 25 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Customization.md:16]: Changed the Color Property content as mentioned above.

File: winui/Shadow/Customization.md (hunk 26 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Customization.md:44]: Changed the Blur Property content as mentioned above.

File: winui/Shadow/Customization.md [hunk 28]
```diff
@@ -0,0 +1,130 @@
...
+   shadow.Content = button;
+
+{% endhiglight %}
+{% endtabs %}
+
+![CornerRadius customization in WinUI Shadow control](Shadow_images/winui_shadow_cornerradius.png)
+
+## Positioning the shadow
+
+The position of shadow can be customized using the `OffsetX` and `OffsetY` properties. The default value of `OffsetX` and `OffsetY` properties is 4.
```
[the remark this answers is not in the document]
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Customization.md:105]: Changed the Offset Property content as mentioned above.

File: winui/Shadow/Getting-Started.md (hunk 27 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Getting-Started.md:4]: Changed the description in Getting-started as mentioned above.

File: winui/Shadow/Getting-Started.md [hunk 29]
```diff
@@ -0,0 +1,165 @@
...
+platform: WinUI
+control: SfShadow
+documentation: ug
+---
+
+# Getting Started with WinUI Shadow
+
+This section explains the steps required to add the WinUI Shadow control and covers only basic features needed to get started with Syncfusion `SfShadow` control.
```
[the remark this answers is not in the document]
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Getting-Started.md:12]: Changed the content as mentioned above.

**Pull request merged**